src/Renderer/LayeredMaterial.js

```diff
@@ -330,22 +329,39 @@ LayeredMaterial.prototype.removeColorLayer = function removeColorLayer(layer) {
     this.uniforms.dTextures_01.value = this.textures[l_COLOR];
 };
 
-LayeredMaterial.prototype.setTexturesLayer = function setTexturesLayer(textures, layerType, layer) {
-    const index = this.indexOfColorLayer(layer);
-    const slotOffset = this.getTextureOffsetByLayerIndex(index);
-    for (let i = 0, max = textures.length; i < max; i++) {
-        if (textures[i]) {
-            if (textures[i].texture !== null) {
-                this.setTexture(textures[i].texture, layerType, i + (slotOffset || 0), textures[i].pitch);
-            } else {
-                this.setLayerVisibility(index, false);
-                break;
+LayeredMaterial.prototype.setLayerTextures = function setLayerTextures(layer, textures) {
```

Comment:
you saw   @mbredif's proposal for the [`LayeredMaterial`](https://github.com/mbredif/itowns/blob/layered_material_rebased/src/Renderer/LayeredMaterial.js)